actioncable/lib/action_cable/subscription_adapter/base.rb

```diff
@@ -25,6 +25,10 @@ def unsubscribe(channel, message_callback)
       def shutdown
         raise NotImplementedError
       end
+
+      def identifier
+        @server.config.cable.fetch(:id, "ActionCable-PID-#{$$}")
```

Comment:
```suggestion
        @server.config.cable[:id] ||= "ActionCable-PID-#{$$}"
```

Don't think `fetch` makes sense here. Either we're going with ^ or `|| "…"`

Think we want to cache this so we don't allocate the string over and over?

Reply:
But what if I want the `id` to be `null`, `fetch` will work, `||=`  not.

Reply:
@kaspth Sorry, I don't think that allocation problem presents here. Server and its corresponding pubsub connection are instantiated once. Anyway, let's change that to be foolproof in the future.